gulpfile.js

```diff
@@ -1,30 +0,0 @@
-var gulp = require("gulp"),
```

Comment:
I don't use this gulpfile, but did you remove it on purpose?

Reply:
Not used. Cool

Reply:
This was added by a team mate who made some contributions to ms-rest-js. He found it useful hence added it. I don't think we are using it. Safe to remove.

Reply:
That's good to know. If we for some reason need it back in the future let's make sure to also add gulp as a devDependency. For now, npm-run-all seems great as it lets us run our node and browser builds and tests in parallel with very little fuss.